src/Driver/Pdo/AbstractPdoCommand.php

```diff
@@ -255,21 +255,13 @@ protected function internalGetQueryResult(int $queryMode): mixed
         return $result;
     }
 
-    /**
-     * Logs the current database query if query logging is on and returns the profiling token if profiling is on.
-     */
-    protected function logQuery(string $rawSql, string $category): void
-    {
-        $this->logger?->log(LogLevel::INFO, $rawSql, [$category]);
-    }
-
     protected function queryInternal(int $queryMode): mixed
     {
         $logCategory = self::class . '::' . $this->getQueryMode($queryMode);
 
         if ($this->logger !== null) {
             $rawSql = $this->getRawSql();
-            $this->logQuery($rawSql, $logCategory);
+            $this->logger->log(LogLevel::INFO, $rawSql, [$logCategory, LogTypes::KEY => LogTypes::TYPE_QUERY]);
```

Comment:
```suggestion
            $this->logger->log(LogLevel::INFO, $rawSql = $this->getRawSql(), [$logCategory, LogTypes::KEY => LogTypes::TYPE_QUERY]);
```

And remove `if ($this->logger !== null) {` condition

Reply:
changed